WcaOnRails/app/models/competition.rb

```diff
@@ -593,6 +610,10 @@ def country
     Country.find_by_id(countryId)
   end
 
+  def continent
+    country ? Continent.find_by_id(country.continentId) : nil
```

Comment:
@larspetrus, I'm not sure I follow. We're talking about `def continent` here, not country.

@jonatanklosko, that's correct, but my point wasn't about the hitting the database so much as it feels weird to rely upon the behavior of `Continent.find_by_id(nil)`. I feel like the current implementation is more straightforward than introducing the `&.` operator.